src/exec/Shelley.hs

```diff
@@ -75,7 +76,7 @@ withShelley rr cdb conf state blockchainTime k = do
 
 -- | Found in cardano-node that the network magic should be 0.
 vData :: NodeToNodeVersionData
-vData = NodeToNodeVersionData { networkMagic = 0 }
+vData = NodeToNodeVersionData { networkMagic = NetworkMagic 0 }
```

Comment:
This won't work anymore, network magic should be taken from the genesis file. Example see https://github.com/input-output-hk/cardano-explorer/pull/102/

Reply:
Is this what you did here? https://github.com/input-output-hk/cardano-byron-proxy/commit/23ec8c0d4d99afd7250adc445b2f77cbe453e286